front-end/src/views/function/components/FunctionDetail.vue

```diff
@@ -0,0 +1,239 @@
+<template>
+  <el-drawer size="40%" @open="onOpen" @close="onClose">
+    <!-- function info -->
+    <el-form ref="info" v-loading="loadingDetail" :rules="rules" style="padding: 0 20px" :model="info"
```

Comment:
Codacy has a fix for the issue: [Replace `·ref="info"·v-loading="loadingDetail"·:rules="rules"·style="padding:·0·20px"` with `⏎······ref="info"⏎······v-loading="loadingDetail"⏎······:rules="rules"⏎······style="padding:·0·20px"⏎·····`](https://app.codacy.com/gh/FunctionStream/function-stream/pullRequest?prid=7834760)
```suggestion
    <el-form
```